charts/sonarqube/ci/openshift-verifier-values.yaml

```diff
@@ -0,0 +1,14 @@
+OpenShift:
+  enabled: false
```

Comment:
Shall we set this to true?

Reply:
If we do, the installation will fail because we are not running the verifier against an Openshift cluster. I'm using the CI cluster.

I think that using an Openshift cluster for this verification is overkill. WDYT?

Reply:
sure but shall we then remove the values from the openshift-verify ones? Openshift is disabled by default...

Reply:
> If we do, the installation will fail because we are not running the verifier against an Openshift cluster. I'm using the CI cluster.
> 
> I think that using an Openshift cluster for this verification is overkill. WDYT?

I think that conceptually it makes sense to run against an openshift cluster considering we are running the openshift specific verifier. I am not sure what tests/verifications are being done so its really an I speak I gut feeling